core/src/odbc_uri.rs

```diff
@@ -44,7 +44,7 @@ lazy_static! {
 }
 
 #[derive(Debug, PartialEq, Eq)]
-pub struct ODBCUri<'a>(HashMap<String, &'a str>);
+pub struct ODBCUri<'a>(pub HashMap<String, &'a str>);
```

Comment:
Let's make this `pub(crate)`. I don't want to leak the abstraction outside of the core crate.

Reply:
Actually, it looks like all you are using this for is to iterate over the keys. I'd just add an iter() method for ODBCUri that calls 0.iter and not make this `pub(crate)` either.